Misha/Files/mp3_tagger/tagger_main.py

```diff
@@ -0,0 +1,45 @@
+import eyed3
+
+
+def input_mp3_file_name(mp3_file_name):
+    file_split = str(mp3_file_name).split('.')
+    if len(file_split) == 1 or file_split[len(file_split)-1] != 'mp3':
+        raise TypeError('Invalid file type')
+    else:
+        try:
+            audiofile = eyed3.load(mp3_file_name)
+            return audiofile
+        except OSError:
```

Comment:
Why do you need this exception raise? Will load function raise it by itself if no file found?

Reply:
raise IOError(f"file not found: {path}")
OSError: file not found: file.mp3

It is an output without my try-except? so its a built-in exception for this error